src/udev/udevadm-verify.c

```diff
@@ -71,12 +83,29 @@ static int parse_argv(int argc, char *argv[]) {
                         if (arg_resolve_name_timing == RESOLVE_NAME_NEVER)
                                 arg_resolve_name_timing = RESOLVE_NAME_LATE;
                         break;
+                case ARG_ROOT:
+                        r = parse_path_argument(optarg, /* suppress_root= */ true, &arg_root);
+                        if (r < 0)
+                                return r;
+                        break;
+
                 case '?':
                         return -EINVAL;
                 default:
                         assert_not_reached();
                 }
 
+        if (arg_root) {
+                if (optind < argc)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                               "Combination of --root= and FILEs is not supported.");
+
+                int fd = open(arg_root, O_CLOEXEC|O_DIRECTORY|O_PATH);
+                if (fd < 0)
+                        return log_error_errno(errno, "open: %s: %m", arg_root);
+                close(fd);
```

Comment:
Please drop it. I guess this is for existence of `arg_root`, but that should not be handled here.